src/Parallel/centrality/stress.jl

```diff
@@ -16,28 +22,45 @@ function distr_stress_centrality(g::AbstractGraph,
     stress = @distributed (+) for s in vs
         temp_stress = zeros(Int64, n_v)
         if degree(g, s) > 0  # this might be 1?
-            state = LightGraphs.dijkstra_shortest_paths(g, s; allpaths=true, trackvertices=true)
+            state = LightGraphs.dijkstra_shortest_paths(
+                g,
+                s;
+                allpaths = true,
+                trackvertices = true,
+            )
             LightGraphs._stress_accumulate_basic!(temp_stress, state, g, s)
         end
         temp_stress
     end
     return stress
 end
 
-function threaded_stress_centrality(g::AbstractGraph,
-    vs::AbstractVector=vertices(g))::Vector{Int64}
+function threaded_stress_centrality(
+    g::AbstractGraph,
+    vs::AbstractVector = vertices(g),
+)::Vector{Int64}
 
     n_v = nv(g)
     k = length(vs)
     isdir = is_directed(g)
 
     # Parallel reduction
-    local_stress = [zeros(Int, n_v) for _ in 1:nthreads()]
+    local_stress = [zeros(Int, n_v) for _ = 1:nthreads()]
```

Comment:
I don't like `=` for ranges. I think `in` reads better. Plus, we have `for v in vertices(g)` all over the place.